zoo-prometheus/prometheus.yml

```
@@ -0,0 +1,24 @@
+global:
```

yamllint

[error] wrong new line character: expected \n

(new-lines)